examples/main.ts

```diff
@@ -0,0 +1,106 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { v, w } from '@dojo/widget-core/d';
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { Registry } from '@dojo/widget-core/Registry';
+
+import { RouteConfig } from './../src/interfaces';
+import { registerRouterInjector } from './../src/RouterInjector';
+import { Link } from './../src/Link';
+import { BasicAppOutlet, BasicAppRouteConfig } from './basic';
+import { UrlParametersAppOutlet, UrlParametersRouteConfig } from './url-parameters';
+import { AmbiguousMatchesOutlet, AmbiguousMatchesRouteConfig } from './ambigious-matches';
+
+const applicationRoutes: RouteConfig[] = [BasicAppRouteConfig, UrlParametersRouteConfig, AmbiguousMatchesRouteConfig];
+
+const registry = new Registry();
+
+registerRouterInjector(applicationRoutes, registry);
+
+const styles = {
+	'text-decoration': 'none',
+	position: 'relative',
+	display: 'block',
+	'line-height': '1.8',
+	cursor: 'auto',
+	color: 'inherit'
+};
+
+class App extends WidgetBase {
```

Comment:
I think you need to add some prettier ignores in here to fix the indentation

Reply:
Wasn't planning on it, it's only examples :)

Reply:
have cleaned `main.ts` up, still uses prettier.